pegasus/sites.v3/code.org/public/js/admins_email_signup_form.js

```diff
@@ -0,0 +1,17 @@
+// Code provided from Google reCAPTCHA
+function timestamp() { var response = document.getElementById("g-recaptcha-response"); if (response == null || response.value.trim() == "") {var elems = JSON.parse(document.getElementsByName("captcha_settings")[0].value);elems["ts"] = JSON.stringify(new Date().getTime());document.getElementsByName("captcha_settings")[0].value = JSON.stringify(elems); } } setInterval(timestamp, 500);
+
+setInterval(timestamp, 500);
+
+// Set a disabled attribute on button until
+// the reCAPTCHA checkbox is clicked
```

Comment:
Uses the `data-callback` attribute on the form's `button` element to check for a successful response — https://developers.google.com/recaptcha/docs/display